docs/docs-new/pages/product/configuration/data-sources/duckdb.mdx

```diff
@@ -0,0 +1,115 @@
+---
+redirect_from:
+  - /config/databases/aws-athena
+---
+
+# DuckDB
+
+[duckdb] is an in-process SQL OLAP database management system, and has support
```

Comment:
Let's also mention that DuckDB can serve as a lightweight alternative to other query engines like AWS Athena or Trino.